skimage/feature/_hog.py

```diff
@@ -61,7 +61,10 @@ def hog(image, orientations=9, pixels_per_cell=(8, 8), cells_per_block=(3, 3),
            For details, see [3]_, [4]_.
 
     visualise : bool, optional
-        Also return an image of the HOG.
+        Also return an image of the HOG.  For each cell and orientation bin,
+        the image contains a line segment that is centered at the cell center,
+        is perpendicular to the bisector of the orientation bin, and has
```

Comment:
Yikes, I completely missed your update here; my apologies.  Yes, that is much better!  But is it simply the midpoint of all the angles, or a weighted midpoint?